src/hooks/use-no-internet-warning-toast.ts

```diff
@@ -0,0 +1,25 @@
+import { useNetInfo } from '@react-native-community/netinfo';
+import { useCallback } from 'react';
+
+import { IconNameEnum } from 'src/components/icon/icon-name.enum';
+import { showWarningToast } from 'src/toast/toast.utils';
+
+export const useNoInternetWarningToast = () => {
```

Comment:
I would rename it to something like `useCallbackIfOnline` and this argument https://github.com/madfish-solutions/templewallet-mobile/pull/1053/files#diff-fabafc1b2a597c3bc13d90da6571b55eb821cb7cff62f8400e6fdf2c7989b2dfR11 to something like `onOnline`